src/components/AppSidebar/AppSidebar.vue

```diff
@@ -204,7 +204,10 @@
 								</form>
 							</template>
 							<!-- secondary title -->
-							<p v-if="subtitle.trim() !== ''" class="app-sidebar-header__subtitle">
+							<p
+								v-if="subtitle.trim() !== ''"
+								v-tooltip.top-start="subtitleTooltip"
```

Comment:
```suggestion
								v-tooltip.auto="subtitleTooltip"
```

Maybe it's best to leave this automatically positioned?

Reply:
Yes, `top-start` looks a bit off. `auto` should be better.